src/views/History.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <n-space vertical :size="30">
+    <PageHeader :module="'history'" :statusOptions="statuses" />
+
+    <n-data-table
+      :columns="columns"
+      :data="logs"
+      :pagination="tablePagination"
+      :render-cell="useRenderCell"
+      :loading="loading"
+      @update:page="handlePageChange"
+      @update:sorter="handleSort"
+    />
+  </n-space>
+</template>
+
+<script setup>
+import ButtonWithPopConfirm from '@/components/Common/ButtonWithPopConfirm';
+import { Icon } from '@iconify/vue';
+import { ref, h, provide } from 'vue';
+import { NAvatar, NSwitch, NTooltip, useMessage } from 'naive-ui';
+import PageHeader from '@/components/Common/PageHeader';
+import moment from 'moment';
+import {
+  useQueryParams,
+  useRenderSortIcon,
+  useRenderCell,
+} from '@/composables';
+
+const statuses = ref([
+  {
+    label: 'Success',
+    value: 'success',
+    icon: 'ep:success-filled',
+    iconColor: '#18A058FF',
+  },
+  {
+    label: 'Failed',
+    value: 'failed',
+    icon: 'ic:round-cancel',
+    iconColor: '#D03050FF',
+  },
+]);
+
+function getChainImg(chain) {
```

Comment:
@anhthichieu this should be in a config file

Reply:
@budtr fixed